admin/src/main/resources/io/buoyant/admin/js/src/combined_client_graph.js

```diff
@@ -89,6 +103,14 @@ define([
           ignoredClients[routerName][client] = false;
         },
 
+        expireClient: function(client) {
+          expiredClients[routerName][client] = true;
```

Comment:
possibly naive question: rather than keeping track of expired clients, is it possible to just remove all references to them?